releasenotes/notes/add-5b9f14b11d8992b9.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: >
+
+features:
+  - |
+    Adds succ_full ansatz option to ``SUCCD`` objects. It can be used in the ``SUCCD`` function,

```

Comment:
Please rephrase as to not mention `succ_full` but actually talk about the new `mirror` keyword argument